api/src/main/java/org/openmrs/module/pihmalawi/setup/MigrateEIDTestResultsSetup.java

```diff
@@ -0,0 +1,40 @@
+package org.openmrs.module.pihmalawi.setup;
+
+import org.apache.commons.lang.time.DateUtils;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.pihmalawi.PihMalawiConstants;
+import org.openmrs.module.pihmalawi.task.MigrateEIDTestResultsTask;
+import org.openmrs.scheduler.SchedulerException;
+import org.openmrs.scheduler.SchedulerService;
+import org.openmrs.scheduler.TaskDefinition;
+
+import java.util.Date;
+
+public class MigrateEIDTestResultsSetup {
+
+    // TODO change this to *remove* this task once migration is complete
```

Comment:
This isn't a critique, but I'm just trying to wrap my head around these technologies -- why didn't you do this with Liquibase?

Reply:
If I use the API I find it a lot easier, I can just deal with the OpenMRS objects and let OpenMRS handle setting the audit fields like date_created, etc, and maintain the proper relationships.  If I'd use liquibase I'd have to figure that all out myself :) and would risk adding faulty data without knowing... 

That being said, liquibase would be *much* faster.